music.py

```diff
@@ -521,6 +541,7 @@ def __init__(self, positions: dict[Hashable, int], guitar: 'Guitar' = None, max_
         # If all fretted notes are >= fret 12, this is a redundant position
         # there is an identical shape 12 frets below that gives (nearly) the same voicing
         self.redundant = all(fret >= 12 for fret in self.positions_dict.values() if fret != 0)
+        self.chord = self.guitar.chord(self.positions_dict)
```

Comment:
Adding a `chord` attr to position allows us to keep track of the unique chord voicings across all the playable positions